apps/src/code-studio/pd/application/teacher1920/Section2ChooseYourProgram.jsx

```diff
@@ -26,6 +26,36 @@ export default class Section2ChooseYourProgram extends LabeledFormComponent {
   }
 
   render() {
+    // This should be kept consistent with the calculation logic in
+    // dashboard/app/models/pd/application/teacher1920_application.rb.
+    const csHowManyMinutes = parseInt(this.props.data.csHowManyMinutes);
+    const csHowManyDaysPerWeek = parseInt(this.props.data.csHowManyDaysPerWeek);
+    const csHowManyWeeksPerYear = parseInt(
+      this.props.data.csHowManyWeeksPerYear
+    );
```

Comment:
Please include the optional radix parameter to `parseInt` to avoid [interpreting numbers with leading zeroes as octal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description).  We should maybe [enable this linter rule](https://eslint.org/docs/rules/radix).